src/SFA.DAS.EmployerFinance.Jobs/SFA.DAS.EmployerFinance.Jobs.csproj

```diff
@@ -21,6 +21,7 @@
 
     <ItemGroup>
         <PackageReference Include="dbup-sqlserver" Version="4.2.0" />
+        <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.5.1" />
```

Comment:
This is the ASP.NET package, I think we should be using `Microsoft.Azure.WebJobs.Logging.ApplicationInsights`.